### PR TITLE
added in java

### PR DESCRIPTION
**Pull request by tf75**

### CODE REVIEW

File: modules/core/Python.md [hunk 1]
```diff
@@ -55,6 +57,25 @@ As well as the above ways to create virtual environments there are two other too
...
+
+```shell
+echo 'eval "$(/opt/homebrew/bin/brew shellenv)"' >> /Users/admin/.zprofile
+eval "$(/opt/homebrew/bin/brew shellenv)"
+```
+
+To install pyspark you will first need to install java.
+
+```shell
+brew install openjdk@11
```
Comment (AdamProbert) [inline modules/core/Python.md:76]: I struggled with 11 on skills for care - had much more luck with jdk@8.
Have you had a different experience?

File: modules/core/Python.md [hunk 2]
```diff
@@ -55,6 +57,25 @@ As well as the above ways to create virtual environments there are two other too
 Poetry in particular has gained popularity due to being able to spin up clean encapsulated environments, easy packaging and quality checking.
 Delta Setup
 
+To install pyspark via homebrew begin with running:
+
+```shell
+/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+```
```
Comment (AdamProbert) [inline modules/core/Python.md:64]: Should mention this is to install homebrew first.
Don't want people running random commands without knowing what they will do

File: modules/core/Python.md (hunk 1 above)
Comment (tf75) [reply modules/core/Python.md:76]: No good point, I change to jdk8

File: modules/core/Python.md (hunk 2 above)
Comment (tf75) [reply modules/core/Python.md:64]: Yeah true if we really going step by step be an idea

**Review by AdamProbert — APPROVED** [review]

**Pull request merged**